logger-cli/src/logger.rs

```diff
@@ -179,6 +183,7 @@ pub fn make_logger(sdk_directory: &Path, config: &StartCommand) -> anyhow::Resul
     ))),
     metadata_provider: Arc::new(LogMetadata {
       timestamp: time::OffsetDateTime::now_utc().into(),
+      ootb_fields: [("_app_version_code".into(), "10".into())].into(),
```

Comment:
this needs to be set in order for lapi to accept the artifact upload, maybe this should be configurable via a cli arg instead?